src/text-editor.coffee

```diff
@@ -2854,23 +2859,27 @@ class TextEditor extends Model
 
   # Essential: Scrolls the editor to the top
   scrollToTop: ->
-    @setScrollTop(0)
```

Comment:
Should we drop deprecated methods from the public API to avoid encouraging further use? Ideally we'd have a *Deprecated* label, but I'm not sure it's worth the time to add it to the docs system at this time.